content/build/cloud/ci.md

```diff
@@ -290,6 +290,43 @@ pipeline {
   }
 }
 ```
+Using the `build.sh` shell script in your `.travis.yml` file under the `install` hook. In this example Travis CI is using Node: 
+
+{{< tabs >}}
+{{< tab name="Travis CI" >}}
+
+
+```yaml
+language: node_js
+node_js:
+  - "14"
```

Comment:
Why is there a node runtime involved here? I guess `minimal` would work given it's ubuntu and has `bash`.

Reply:
Hi @dvdksn,

True, I guess as stated in original PR it was just an example using Node. I appreciate the feedback!